app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,13 @@
+<header class="navbar navbar-fixed-top navbar-custom">
+  <%= link_to "private events app", root_path, id: "logo" %>
+  <ul class="nav navbar-nav navbar-right">
+    <% if signed? %>
+      <li class="user-btn "><%= link_to "#{current_user.name.split(" ")[0]} Events" , user_path(current_user) %></li>
+      <li><%= link_to "New Event", new_event_path %></li>
+      <li><%= link_to "Logout", logout_path %></li>
+    <% else %>
+      <li><%= link_to "Sign up", new_user_path %></li>
+      <li><%= link_to "Sign in", signin_path %></li>
+    <% end %>
```

Comment:
* [ ] There should be absolutely NO logic in views. [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright